gmplot/gmplot.py

```diff
@@ -45,12 +46,15 @@ def geocode(self, location_string):
     def grid(self, slat, elat, latin, slng, elng, lngin):
         self.gridsetting = [slat, elat, latin, slng, elng, lngin]
 
-    def marker(self, lat, lng, color='#FF0000', c=None, title="no implementation"):
+    def marker(self, lat, lng, color='#FF0000', c=None, title="no implementation", label="no implementation"):
         if c:
             color = c
         color = self.color_dict.get(color, color)
         color = self.html_color_codes.get(color, color)
-        self.points.append((lat, lng, color[1:], title))
+        self.points.append((lat, lng, color[1:], title, label))
+
+    def directions(self, orig, dest, waypoints):
+        self.direction_points.append([orig, dest, waypoints]);
```

Comment:
Shouldn't this be a tuple (`(orig, dest, waypoints)`) to match the other functions?